core/os/os.cpp

```diff
@@ -514,6 +514,10 @@ bool OS::has_feature(const String &p_feature) {
 	if (p_feature == "wasm") {
 		return true;
 	}
+#elif defined(__loongarch__)
+	if (p_feature == "loongarch64") {
+		return true;
+	}
```

Comment:
Same here, we can use the macro to be more specific about architecture feature flags.

```suggestion
#elif defined(__loongarch__)
#if defined(__loongarch64)
	if (p_feature == "loongarch64") {
		return true;
	}
#endif
	if (p_feature == "loongarch") {
		return true;
	}
```